save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/HistoryView.kt

```diff
@@ -85,19 +92,35 @@ external interface HistoryViewState : State {
      * Label of confirm Window
      */
     var confirmLabel: String
+
+    /**
+     * All filters in one value [filters]
+     */
+    var filters: ExecutionFilters
+}
+
+/**
+ * [Props] of a data table with filters for execution
+ */
+external interface FiltersProps<D : Any> : TableProps<D> {
```

Comment:
If it expects `ExecutionFilters`, maybe `D` should be Execution\ExecutionDto?

Reply:
Fixed